src/sentry/static/sentry/app/components/shareIssue.tsx

```diff
@@ -0,0 +1,232 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import {IconCopy, IconRefresh} from 'app/icons';
+import space from 'app/styles/space';
+import AutoSelectText from 'app/components/autoSelectText';
+import Button from 'app/components/button';
+import Clipboard from 'app/components/clipboard';
+import Confirm from 'app/components/confirm';
+import DropdownLink from 'app/components/dropdownLink';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import SpreadLayout from 'app/components/spreadLayout';
+import Switch from 'app/components/switch';
+
+type ContainerProps = {
+  shareUrl: string;
+  onConfirming: () => void;
+  onConfirm: () => void;
+  onCancel: () => void;
+};
+
+class ShareUrlContainer extends React.Component<ContainerProps> {
+  urlRef?: AutoSelectText | null;
```

Comment:
you are using this type in two different places, maybe you could create a common type and just reuse :)